cmd/codebuild/reset/main.go

```diff
@@ -152,11 +148,11 @@ func nukeAccount(svc *service, isDryRun bool) error {
 
 	// Configure the NukeAccountInput
 	nukeAccountInput := reset.NukeAccountInput{
-		AccountID:  config.accountID,
-		RoleName:   config.accountAdminRoleName,
+		AccountID:  conf.AccountID,
+		RoleName:   conf.AdminRoleName,
 		ConfigPath: configFile,
-		NoDryRun:   !isDryRun,
-		Token:      svc.tokenService(),
+		DryRun:     !conf.IsNukeEnabled,
```

Comment:
Changed `NoDryRun` --> `DryRun`. It was way to hard to follow the double negatives here